CMakeLists.txt

```diff
@@ -73,6 +73,17 @@ set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/out/lib")
 set(MLT_MODULE_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/out/lib/mlt")
 set(MLT_DATA_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/out/share/mlt")
 
+# https://gitlab.kitware.com/cmake/community/-/wikis/doc/cmake/RPATH-handling
+set(CMAKE_SKIP_BUILD_RPATH FALSE)
+set(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE)
+set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib")
+set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
+list(FIND CMAKE_PLATFORM_IMPLICIT_LINK_DIRECTORIES "${CMAKE_INSTALL_PREFIX}/lib" isSystemDir)
+if("${isSystemDir}" STREQUAL "-1")
+    set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib")
+endif("${isSystemDir}" STREQUAL "-1")
```

Comment:
I know this is based on code from the cmake wiki, but a fully specified `endif` is inconsistent with the rest of the CMakeLists.txt files, and I am not a fan of it. The [docs](https://cmake.org/cmake/help/latest/command/endif.html) say it is only for backwards compatibility, but we specify a minimum CMake version; so, it is not needed.

Besides that, I still need to test more if this change has any weird side effect on Windows, MacPorts, or Shotcut's relocatable builds. That is kind of a pain. For some of this, I might end up accepting and then restricting if needed.